# Μαθηματικές Εξισώσεις
Υπολογίστε τις ακόλουθες εκφράσεις σε Python και επαληθευστε το αποτέλεσμα.

## Παράδειγμα
## Υπολόγισε την ακόλουθη έκφραση στην Python:$$ (5 + 3) \times 2^3 - \frac{40}{2 + 3} $$

In [1]:
(5+3)*(2**3) - (40//(2+3))

56

## Άσκηση 1

$$ (23 + 12) \times 5^3 - \frac{120}{7 + 2} $$

In [2]:
(23 + 12) * 5**3 - 120/(7 + 2)

4361.666666666667

### Αποτέλεσμα: **4361.666667**

## Άσκηση 2

$$ ((45 - 17)(9 - 3 \times 2))^2 $$

In [3]:
((45 - 17) * (9 - 3*2))**2

7056

### Αποτέλεσμα: **7056**

## Άσκηση 3

$$ \frac{9^4}{50 \bmod 6} $$

In [4]:
9**4 / (50 % 6)

3280.5

### Αποτέλεσμα: **3280.5**

## Άσκηση 4

$$ (50 + 22) \times \frac{16}{5 - 2.5} $$

In [ ]:
(50 + 22) * 16 / (5 - 2.5)

460.8

### Αποτέλεσμα: **460.8**

## Άσκηση 5

$$ (8 \times 7)^3 - \frac{250}{3 + 4\times 2} $$

In [6]:
(8 * 7)**3 - 250 / (3 + 4*2)

175593.27272727274

### Αποτέλεσμα: **175593.272727**

## Άσκηση 6

$$ \frac{32}{4}(15 - 5) + 4^5 $$

In [39]:
(32//4) * (15 - 5) + 4**5

1104

### Αποτέλεσμα: **1104**

## Άσκηση 7

$$ (72 - 3\times 8) \times 10^2 \\div (13 - 4) $$

In [8]:
(72 - 3*8) * 10**2 / (13 - 4)

533.3333333333334

### Αποτέλεσμα: **533.333333**

## Άσκηση 8

$$ \frac{95 + 40}{32 \bmod 7} + 5^4 $$

In [9]:
(95 + 40) / (32 % 7) + 5**4

658.75

### Αποτέλεσμα: **658.75**

## Άσκηση 9

$$ \left(40 \times 3 + \frac{68}{5 - 2}\right)^3 $$

In [10]:
(40 * 3 + 68 / (5 - 2))**3

2903805.629629629

### Αποτέλεσμα: **2903805.62963**

## Άσκηση 10

$$ ((18 \times 4) - \tfrac{36}{3})^2 + 7\times 5 - 12^3 $$

In [38]:
((18 * 4) - 36//3)**2 + 7*5 - 12**3

1907

### Αποτέλεσμα: **1907**

## Άσκηση 11

$$ (14 + 26) \times 3^2 - \frac{180}{7 + 8} $$

In [37]:
(14 + 26) * 3**2 - 180 // (7 + 8)

348

### Αποτέλεσμα: **348**

## Άσκηση 12
$$ \big((50 - 17)(8 + 4) - \tfrac{36}{3}\big)^2 $$

In [36]:
((50 - 17) * (8 + 4) - 36//3)**2

147456

### Αποτέλεσμα: **147456**

## Άσκηση 13

$$ \frac{9^3 - 4^4}{73 \bmod 9} $$

In [35]:
(9**3 - 4**4) // (73 % 9)

473

### Αποτέλεσμα: **473**

## Άσκηση 14

$$ \frac{\big(3(18 - 5)\big)^3}{2 + 7} $$

In [34]:
(3 * (18 - 5))**3 // (2 + 7)

6591

### Αποτέλεσμα: **6591**

## Άσκηση 15

$$ 7^5 - 6^4 + \frac{48}{6}(21 - 13) $$

In [33]:
7**5 - 6**4 + (48//6) * (21 - 13)

15575

### Αποτέλεσμα: **15575**

## Άσκηση 16

$$ \frac{96}{5 + 7} \times \big((15 - 9)\cdot 4\big) - 2^8 $$

In [32]:
96 // (5 + 7) * ((15 - 9) * 4) - 2**8

-64

### Αποτέλεσμα: **-64**

## Άσκηση 17

$$ \big(35\times 4 - \tfrac{28}{2}\big)^3 $$

In [31]:
(35 * 4 - 28//2)**3

2000376

### Αποτέλεσμα: **2000376**

## Άσκηση 18

$$ \frac{(11 + 9)(13 - 7)\times 5^2}{4 + 1} $$

In [30]:
(11 + 9) * (13 - 7) * 5**2 // (4 + 1)

600

### Αποτέλεσμα: **600**

## Άσκηση 19

$$ 2^{10} - \big(3^5 - 2^6\big) $$

In [20]:
2**10 - (3**5 - 2**6)

845

### Αποτέλεσμα: **845**

## Άσκηση 20

$$ \big(\tfrac{64}{8} + \tfrac{81}{9}\big)^2 - 7(5 + 6) $$

In [29]:
(64//8 + 81//9)**2 - 7*(5 + 6)

212

### Αποτέλεσμα: **212**

## Άσκηση 21

$$ \frac{(27 - 5\times 3)(14 + 2)}{(19 \bmod 5) + 1} $$

In [22]:
(27 - 5*3) * (14 + 2) / ((19 % 5) + 1)

38.4

### Αποτέλεσμα: **38.4**

## Άσκηση 22

$$ \frac{10^3}{(6 + 4)(3 + 2)} + 17 $$

In [28]:
10**3 // ((6 + 4) * (3 + 2)) + 17

37

### Αποτέλεσμα: **37**

## Άσκηση 23

$$ \big(9\times 7 - 4\times 8\big)^2 + \frac{3^6}{2^3} $$

In [24]:
(9*7 - 4*8)**2 + 3**6 / 2**3

1052.125

### Αποτέλεσμα: **1052.125**

## Άσκηση 24
### Μαθηματική Μορφή
$$ \frac{(5 + 11)(7 + 9) - 3^4}{(8 - 5)\times 3} $$

In [25]:
((5 + 11) * (7 + 9) - 3**4) / ((8 - 5) * 3)

19.444444444444443

### Αποτέλεσμα: **19.444444**

## Άσκηση 25

$$ \big(4^5 - 2^7 + 3\cdot 19\big)  \Big(\tfrac{12}{3} + \tfrac{18}{6}\Big) $$

In [27]:
(4**5 - 2**7 + 3*19) * (12//3 + 18//6)

6671

### Αποτέλεσμα: **6671**